File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/source/remote/response/VideosResponse.java
package com.ryanrvldo.moviecatalogue.data.source.remote.response;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;
import com.ryanrvldo.moviecatalogue.data.source.model.Video;

import java.util.List;

import lombok.Data;

@Data
public class VideosResponse {
    @SerializedName("results")
    @Expose
    private List<Video> videos;
}

File: app/src/main/java/com/ryanrvldo/moviecatalogue/data/remote/response/VideosResponse.kt
package com.ryanrvldo.moviecatalogue.data.remote.response

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import com.ryanrvldo.moviecatalogue.data.model.Video

data class VideosResponse(
    @SerializedName("results") @Expose val videos: List<Video>
)